src/Symfony/Bundle/FrameworkBundle/Resources/config/templating_php.xml

```diff
@@ -55,9 +55,8 @@
             <argument /> <!-- version format -->
         </service>
 
-        <service id="templating.asset.request_aware_package" class="Symfony\Component\Templating\Asset\PackageInterface" abstract="true">
-            <factory service="templating.asset.package_factory" method="getPackage" />
-            <argument type="service" id="request" strict="false" />
+        <service id="templating.asset.request_aware_package" class="Symfony\Component\Templating\Asset\PackageInterface" factory-service="templating.asset.package_factory" factory-method="getPackage" abstract="true">
+            <argument type="expression">service('request_stack').getMasterRequest()</argument>
```

Comment:
This will break if the master request is not available (in the CLI for instance). It would be better to inject the RequestStack itself.